run.sh

```diff
@@ -2,13 +2,33 @@
 
 names="deriv gf kin conc watch bbcore vecplay patstim"
 
-set -e
+for i in $names ; do
+  mkdir -p test${i}dat
+done
+
+for i in $names ; do
+  ./x86_64/special -c sim_time=100 test${i}.hoc
```

Comment:
Could we run this mpi? e.g. mpirun or srun? that way we can check if parallel run is also working fine.